addons/mail/static/src/components/activity_box/activity_box.scss

```diff
@@ -14,13 +14,17 @@
     padding: map-get($spacers, 0) map-get($spacers, 2);
 }
 
+.o_ActivityBox_titleBadges {
+    margin-inline-end: map-get($spacers, 3);
+}
+
 .o_ActivityBox_titleLine {
     flex: 1 1 auto;
     width: auto;
 }
 
 .o_ActivityBox_titleText {
-    padding: map-get($spacers, 0) map-get($spacers, 3);
+    margin: map-get($spacers, 0) map-get($spacers, 3);
```

Comment:
I still find it strange that the element on the left is responsible for the left margin AND the margin in the middle, while the right element is only responsible of the margin on the right.

I guess since the title is always there it shouldn't be an issue, but if we some day we decide to hide the title in some situations, the left margin of the badge will be wrong. Just to keep in mind.